ui/app/helpers/options-for-backend.js

```diff
@@ -87,6 +87,7 @@ const SECRET_BACKENDS = {
     displayName: 'Transformation',
     navigateTree: false,
     listItemPartial: 'secret-list/transform-list-item',
+    message: 'create a transformation and a role',
```

Comment:
small suggestion: `message` is a bit vague and couple mean anything. Maybe we call this `firstStep` instead?

Reply:
@hashishaw - sounds good! thanks for the suggestion!